src/utils/index.ts

```
@@ -18,14 +19,61 @@ const extensions = (Module as any)._extensions;
+
+  // capture the stack
+  const obj: any = {};
+  Error.captureStackTrace(obj);
+  let callSite = obj.stack[stackIndex];
+  let fileName = '';
+  if (callSite) {
+    // egg-mock will create a proxy
+    // https://github.com/eggjs/egg-mock/blob/master/lib/app.js#L174
+    fileName = callSite.getFileName();
+    /* istanbul ignore if */
+    if (fileName && fileName.endsWith('egg-mock/lib/app.js')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)